### ISSUE TITLE
[Feature Request] ability to close documents

### ISSUE THREAD

**Issue by BasilTomato** [cryptee/web-client#110]
**Is your feature request related to a problem? Please describe.**
It would be useful to be able to close an opened document after the modifications are completed but no other document is opened yet.

**Describe the solution you'd like**
Ability to "close" a given document in the document context menu, as it was possible in 2.0.
After the document is closed the "welcome to cryptee.." screen may be shown.

**Additional context**
n/a

**Comment by johnozbay** [comment]
Hi there! 

After ~2 years of having this feature, we realized this feature caused more confusion than it solved problems. In general, none of the other applications our users moved away from had a "close" feature, and this confused the hell out of everyone. Think Evernote / Google Docs etc. none of them have something like this per-se. 

So for some reason, as silly as it may sound like, our support team got at least 5 questions every week like : "if I close this document will it be deleted?" – "does close document mean like close it in an envelope / hide or? like ghosting?" etc. 

So to prevent confusion we intentionally removed this feature altogether.
We figured there are most likely two reasons why someone would want to close a document: 

a) if they want to open another document / create a new one. In that case, it's quicker to open the document / create it. 

b) If they're going to step away from their computers and don't want Cryptee to be open – in this case, the better / safer thing to do is to close Cryptee, and we figured it's best to encourage this instead. 

We are of course open for ideas and would be happy to hear everyone's thoughts on this – but unless there's a very specific use case for this – we won't be adding this feature back again.
 
I hope these make sense! ✌🏻
